elide-datastore/elide-datastore-hibernate/src/test/java/com/yahoo/elide/datastores/hibernate/hql/SubCollectionFetchQueryBuilderTest.java

```diff
@@ -112,6 +118,78 @@ public void testSubCollectionFetchWithSorting() {
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testSubCollectionFetchWithIncludedToManyRelation() {
```

Comment:
We should probably add at least one test where we do a FETCH JOIN when getting a sub collection

Reply:
I would add another test with INcludedToOneRelationship

Reply:
@aklish As per my understanding this test should serve the purpose.
https://github.com/yahoo/elide/pull/1459/files#diff-d507c81b32b694bbf433353645234239R118
Please let me know if I'm not understanding the comment.